README.md

```diff
@@ -20,6 +20,33 @@ What's this? This is a template repository that sets up a few minor systems for
 
 ## FAQs
 
+### Some lint is unnecessary :( How do I disable it?
+There are actually 2 main ways to disable lint. Disabling the "rule" entirely, or just a single instance/in a file!
```

Comment:
Not sure if I’m misunderstanding something or typo for the slash in 'instance/in'. (Probably isn’t typo since used more than once so I guess maybe just an easier way to understand)